src/main/java/com/libtask/library2/services/BookService.java

```diff
@@ -0,0 +1,13 @@
+package com.libtask.library2.services;
+
+
+import com.libtask.library2.repositories.BookRepository;
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@Service
+@AllArgsConstructor

```

Comment:
user RequiredArgsConstructor to prevent property injection problems in future

Reply:
still don`t use @RequiredArgsConstructor

Reply:
this is how the problem was solved in the controller, but this is already irrelevant, but the annotation remains